ChangeLog.d/ecp-keypair-utilities.txt

```diff
@@ -0,0 +1,3 @@
+Features
+   * Add utility functions to manipulate mbedtls_ecp_keypair objects, filling
```

Comment:
Minor: I think it would help for the ChangeLog to name the functions that were added, in order to make it easier for users to discover them.